python/lsst/afw/fits/SConscript

```diff
@@ -1,3 +1,3 @@
 ## -*- python -*-
 #from lsst.sconsUtils import scripts
-#scripts.BasicSConscript.python(['fitsLib'])
+#scripts.BasicSConscript.pybind11(['fits'])
```

Comment:
I still think this deserves a comment in the code describing why it's commented out (so a user has a clue what might be needed to uncomment it or delete it). It is more obscure than the boilerplate pybind11 includes that are commented out.